### PR TITLE
feature(integration): add integration templates for zoho-mail

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe your changes

I have added integration templates for zoho-mail. These include templates to sync emails and templates action to add a user to an organization and send an email.

### CODE REVIEW

File: packages/shared/flows.yaml [hunk 1]
```diff
@@ -2719,6 +2719,37 @@ integrations:
                 endpoint: /zoho-mail/tasks
                 scopes:
                     - ZohoMail.tasks.READ
+            zoho-mail-emails:
+                runs: every 6 hours
+                description: |
+                    Fetches a list of all your account's emails in Zoho mail
+                metadata:
+                    description: |
+                        We need to save a an accountId to track and update emails' synchronizations
```
Comment (khaliqgant) [inline packages/shared/flows.yaml]: fix typo

File: integration-templates/zoho-mail/zoho-mail-emails.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+const LIMIT = 100;
+
+export default async function fetchData(nango: NangoSync) {
+    let totalRecords = 0;
+    let offset = 1;
+
+    const metadata = (await nango.getMetadata()) || {};
+    const accountId = metadata['accountId'] ? String(metadata['accountId']) : '';
+
```
Comment (khaliqgant) [inline integration-templates/zoho-mail/zoho-mail-emails.ts:11]: Shouldn't this sync bail if there is no `accountId`?

Comment (khaliqgant) [reply integration-templates/zoho-mail/zoho-mail-emails.ts:11]: Is the accountId not apart of the nango metadata? Where does the accountId come from?

Comment (hassan254-prog) [reply integration-templates/zoho-mail/zoho-mail-emails.ts:11]: @khaliqgant Yes, the sync should fail if there is no accountId. I have updated my pull request. Thanks.

File: integration-templates/zoho-mail/nango.yaml [hunk 3]
```diff
@@ -1,47 +1,124 @@
...
+                output: ZohoMailTask
+                sync_type: full
+                endpoint: /zoho-mail/tasks
+                scopes:
+                    - ZohoMail.tasks.READ
+            zoho-mail-emails:
+                runs: every 6 hours
+                description: |
+                    Fetches a list of all your account's emails in Zoho mail
+                metadata:
```
Comment (khaliqgant) [inline integration-templates/zoho-mail/nango.yaml]: This extra metadata object should be removed

Comment (hassan254-prog) [reply integration-templates/zoho-mail/nango.yaml]: @khaliqgant , just updated my pull request. Quick one, when should we add the metadata object?

Comment (khaliqgant) [reply integration-templates/zoho-mail/nango.yaml]: You shouldn't really with the updated v2 version of the nango.yaml

**Comment by khaliqgant** [comment]
@hassan254-prog can you resolve conflicts with this PR?

File: integration-templates/zoho-mail/nango.yaml (hunk 3 above)
Comment (hassan254-prog) [reply integration-templates/zoho-mail/nango.yaml]: @khaliqgant thanks for the clarification 🙏.

**Comment by hassan254-prog** [comment]
@khaliqgant just resolved conflicts on this and all test passed 🚀

**Review by khaliqgant — APPROVED** [review]

**Pull request merged**